client/src/components/ProfileSettings/Payment/Payment.tsx

```diff
@@ -1,7 +1,112 @@
+import { useState } from 'react';
+import useStyles from './useStyles';
+// the following images will be replaced with card snapshots
+import image1 from '../../../Images/credit-card-1369111_1280.png';
+import image2 from '../../../Images/credit-cards-1583534_1920.jpg';
+import {
+  Button,
+  Card,
+  Typography,
+  TextField,
+  CardActions,
+  Paper,
+  Grid,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+} from '@material-ui/core';
+
 const Payment = (): JSX.Element => {
-    return(
-        <h1>This is Payment tab</h1>
-    );
-}
+  const classes = useStyles();
+  const [open, setOpen] = useState(false);
+
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
+    try {
+      //make post request and save card details
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Paper elevation={5} className={classes.paymentPaper}>
+      <Typography variant="h5" className={classes.paymentHeader}>
+        Payment Methods
+      </Typography>
+      <Card className={classes.paymentCard}>
+        <Typography variant="subtitle1" className={classes.paymentText}>
+          Saved Payment Profiles:
+        </Typography>
+        <Grid container className={classes.customerCard}>
+          <Grid item sm={6}>
+            <Paper elevation={1} className={classes.customerSingleCard}>
+              <img src={image1} className={classes.paymentImage} />
+            </Paper>
+          </Grid>
+          <Grid item sm={6}>
+            <Paper elevation={1} className={classes.customerSingleCard}>
+              <img src={image2} className={classes.paymentImage} />
+            </Paper>
+          </Grid>
+        </Grid>
+        <CardActions>
+          <form action="/" method="POST" className={classes.form} onSubmit={handleSubmit}>
```

Comment:
This `form` should be within the dialog, further below.  We also don't need the `action` and `method` attributes since we handle submission using JS.